### PR TITLE
Closes #206 Removed author from programming stratergy.

### LINKED ISSUE #206: Remove need for `@author` tag from Programming Strategy

### PR DESCRIPTION
**Pull request by DeclanHodges**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiraldev/devel/articles/development_process.html) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `devel` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#deprecation)?
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/devel/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiraldev/devel/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately 
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Review by bms63 — CHANGES_REQUESTED** [review]
Could you put in a note for the NEws that we are no longer tracking authors in the code scripts, but in the Description please

**Comment by DeclanHodges** [comment]
> Could you put in a note for the NEws that we are no longer tracking authors in the code scripts, but in the Description please

Hi Ben, I'm not sure what you meant by "but in the description", but I've added a note to the news. Is this okay?

File: DESCRIPTION [hunk 1]
```diff
@@ -27,7 +27,7 @@ Encoding: UTF-8
 Language: en-US
 LazyData: false
 Roxygen: list(markdown = TRUE)
-RoxygenNote: 7.2.1
```
Comment (bms63) [inline DESCRIPTION:30]: @DeclanHodges this should not be changing.  We need to use the roxygen in the lock file

File: NEWS.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ to make the default behavior consistent. (#87)
 
 ## Documentation
   - New section in programming strategy regarding comments (#71)
+  - Removed requirement to add `@author` tags to code scripts from programming 
+  strategy, as we will no longer be tracking authors. (#206)
```
Comment (bms63) [inline NEWS.md:24]:
```suggestion
  - Removed requirement to add `@author` tags to code scripts from programming 
  strategy, as we will only be tracking authors in the DESCRIPTION file. (#206)
```

**Review by bms63 — CHANGES_REQUESTED** [review]
Looking good just a few minor adjustments.  Once this is over then can you start removing authors from all the `admiral` files.

File: NEWS.md (hunk 2 above)
Comment (DeclanHodges) [reply NEWS.md:24]: Ah I see what you meant by your original comment now, thanks.

**Comment by DeclanHodges** [comment]
Hi @bms63, I've made the changes, so this PR should be good to go.

**Review by bms63 — APPROVED** [review]
Thanks!!

**Pull request merged**